File: app/src/main/java/facebook/com/socialrunner/domain/data/entity/Runner.java
package facebook.com.socialrunner.domain.data.entity;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Runner extends Entity {

    private String name;
    private List<String> routeIds;
}

File: app/src/main/java/facebook/com/socialrunner/domain/data/entity/Runner.kt
package facebook.com.socialrunner.domain.data.entity


data class Runner(
        val name: String = "",
        var routeIds: MutableList<String> = arrayListOf(),
        val pace: Double = 0.0,
        val longitude: Double = 0.0,
        val latitude: Double = 0.0,
        val routeID: String = "",
        val isRunning: Boolean = false
) {
    fun position(): Position = Position(longitude = longitude, latitude = latitude)
}